badgecheck/tasks/validation.py

```diff
@@ -1,19 +1,22 @@
 import aniso8601
+from datetime import datetime
+from dateutil.tz import tzutc
```

Comment:
Should we use `pytz` instead of `dateutil` as, all we're using it for is UTC tz info? (see http://www.assert.cc/2014/05/25/which-python-time-zone-library.html)